UVAtlasTool/Mesh.h

```diff
@@ -12,11 +12,11 @@
 
 #include <Windows.h>
 
+#include <cstdint>
 #include <memory>
+#include <ostream>
 #include <string>
-#include <vector>
 
-#include <stdint.h>
```

Comment:
deprecated header, changed it to suggested <cstdint>

Reply:
That's fine. I have a habit of using ``#include <stdint.h>`` and ``#include <cmath>`` because of some quirky internal code scenarios where using ``#include <cstdint>`` or ``#include <cmath>`` caused problems.

For this codebase, C++ is assumed.